src/components/swapv2/LimitOrder/Modals/ConfirmOrderModal.tsx

```diff
@@ -41,9 +44,12 @@ export default memo(function ConfirmOrderModal({
   marketPrice: BaseTradeInfo | undefined
   rateInfo: RateInfo
   note?: string
+  warningMessage: ReactNode[]
 }) {
   const { account } = useActiveWeb3React()
 
+  const displayCurrencyOut = currencyOut?.isNative ? currencyOut.wrapped : currencyOut
```

Comment:
Look like this will cause infinity loop.

Reply:
why bro

Reply:
`currencyOut.wrapped` returns new instance everytime it called, then it trigger hooks to run which might causing infinity loop rendering

Reply:
because it is a object, and it is used in the useMemo below